src/ck-core/ckfutures.C

```diff
@@ -426,6 +426,23 @@ public:
   }
 };
 
+// TODO :
+// a poorly located helper function for getting a buffer from a message
+// should be moved elsewhere (when PR#3228 is merged), and probably named ck::get_message_buffer(void*)
+void* CkGetMsgBuffer(void* msg) {
+  auto env = UsrToEnv(msg);
```

Comment:
Should you so choose:
```suggestion
  auto* env = UsrToEnv(msg);
```